macros/edr/tests/on_run_end/handle_tests_results.sql

```diff
@@ -275,23 +276,27 @@
         'test_name': elementary.insensitive_get_dict_value(test_node, 'short_name'),
         'test_params': elementary.insensitive_get_dict_value(test_node, 'test_params'),
         'severity': elementary.insensitive_get_dict_value(test_node, 'severity'),
-        'status': elementary.insensitive_get_dict_value(run_result_dict, 'status')
+        'status': elementary.insensitive_get_dict_value(run_result_dict, 'status'),
+        'failures': elementary.insensitive_get_dict_value(run_result_dict, 'failures')
     } %}
     {{ return(test_result_dict) }}
 {% endmacro %}
 
 {% macro get_schema_change_test_result(run_result_dict, schema_change_dict, test_node) %}
     {% set test_result_dict = schema_change_dict %}
+    {# Becasue we use compiled sql for fetching data sampling, we currently not using the macro get_model_sql_from_node #}
```

Comment:
I think this comment is not relevant as the macro is called get_model_sql_from_node and here it's a test